TTS/tts/utils/speakers.py

```diff
@@ -187,6 +194,20 @@ def get_d_vectors_by_speaker(self, speaker_idx: str) -> List[List]:
         """
         return [x["embedding"] for x in self.d_vectors.values() if x["name"] == speaker_idx]
 
+    def get_d_vector_by_speaker(self, speaker_idx: str) -> np.ndarray:
```

Comment:
This returns the first matching d-vector of the speaker right

Reply:
If so if you set num_samples=1 in the function below it is the same thing

Reply:
Done